src/main/java/io/temporal/internal/replay/ExecuteLocalActivityParameters.java

```diff
@@ -24,19 +24,17 @@
 public class ExecuteLocalActivityParameters {
 
   private final PollActivityTaskQueueResponse.Builder activityTask;
-  private final long elapsedTime;
 
-  public ExecuteLocalActivityParameters(
-      PollActivityTaskQueueResponse.Builder activityTask, long elapsedTime) {
+  public ExecuteLocalActivityParameters(PollActivityTaskQueueResponse.Builder activityTask) {
     this.activityTask = activityTask;
-    this.elapsedTime = elapsedTime;
   }
 
   public PollActivityTaskQueueResponse.Builder getActivityTask() {
     return activityTask;
   }
 
-  public long getElapsedTime() {
-    return elapsedTime;
+  @Override
+  public String toString() {
+    return "ExecuteLocalActivityParameters{" + "activityTask=" + activityTask + '}';
```

Comment:
[Nit] no single or double quote delimiters around activityTask here?

Reply:
Also you can remove the first plus sign and concatenate the first two strings (e.g. "ExecuteLocalActivityParameters{activityTask=" + activityTask + "}"